.env.example

```diff
@@ -115,3 +121,23 @@ AIRFLOW_SERVICE_SCHEME=http
 AIRFLOW_SERVICE_HOST=airflow-webserver
 AIRFLOW_SERVICE_PORT=8080
 AIRFLOW_SERVICE_PATH_PREFIX=/api/v1
+
+# Search Service
+ES_HOST=badgerdoc-elasticsearch
+ES_PORT=9200
+APP_TITLE=Badgerdoc Search
+ANNOTATION_URL=http://annotation
+ANNOTATION_CATEGORIES=/categories
```

Comment:
You can move URI inside search microservice, we really don't need to configure path for all services globally

Reply:
Ok, I'd agree. Am I right to put in  .env for microservice is enough?